gulpfile.js

```diff
@@ -379,8 +379,20 @@ function dist_src() {
 }
 
 function dist_node_modules_css() {
-  return gulp
-    .src("./node_modules/**/*.min.css")
+    const platforms = getPlatforms();
+    const isAndroid = platforms.includes('android');
+
+    const cssSources = [
+        './node_modules/**/*.min.css',
+    ];
+
+    if (isAndroid) {
+        cssSources.push("./node_modules/**/*.woff2");
+        cssSources.push("./node_modules/**/*.ttf");
+    }
+
+    return gulp
```

Comment:
```suggestion
  return gulp
```

Reply:
4 space indentation is correct.

Reply:
Then the two lines below miss 4 intendations but no worries 😃